build.gradle

```diff
@@ -73,4 +75,12 @@ subprojects {
   afterEvaluate {
     tasks.findByName('check').dependsOn('checkstyle')
   }
-}
\ No newline at end of file
+
+  if (isCi) {
```

Comment:
Let's just always enable this. I only do conditional on projects that might be affected by the slower compiles.